contracts/svm/programs/express_relay/src/token.rs

```diff
@@ -9,19 +9,38 @@ use {
     },
 };
 
+pub fn get_token_account_checked(
+    account: &AccountInfo,
+    expected_program_id: &Pubkey,
+) -> Result<TokenAccount> {
+    if account.data_len() == 0 {
+        return Err(ErrorCode::AccountNotInitialized.into());
```

Comment:
this check is not needed because `try_deserialize` will fail (i understand if you want to keep it)